recipes/laspy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "laspy" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8768499aa429c028debdff67aa738bcf323ee777025528b842f36cc22d4ccf8d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
You have some entry-points, let's add them here and add tests for them:

```yaml
  entry_points:
    - lascopy = laspy.tools.lascopy:main
    - lasexplorer = laspy.tools.lasexplorer:main
    - lasnoise = laspy.tools.lasnoise:main
    - lasverify = laspy.tools.lasverify:main
    - lasvalidate = laspy.tools.lasvalidate:main
    - lasviewer = laspy.tools.lasviewer:main
```

A test can be a an entry in the the `test/commands` invoking the help or something that returns `0` but tests the existence of the CLI:

```
- lasviewer -h
```

Reply:
Thanks, that seemed to work!